services/src/main/java/org/keycloak/protocol/oid4vc/issuance/signing/SdJwtSigningService.java

```diff
@@ -31,12 +48,105 @@
  */
 public class SdJwtSigningService extends SigningService<String> {
 
-    public SdJwtSigningService(KeycloakSession keycloakSession, String keyId, String algorithmType) {
+    private static final Logger LOGGER = Logger.getLogger(SdJwtSigningService.class);
+
+    private static final String ISSUER_CLAIM ="iss";
+    private static final String NOT_BEFORE_CLAIM ="nbf";
+    private static final String VERIFIABLE_CREDENTIAL_TYPE_CLAIM = "vct";
+    private static final String SELECTIVE_DISCLOSURE_ALGORITHM_CLAIM = "_sd_jwt";
```

Comment:
Line can be remove, unless we want to embed the sdJWt into another json document.
```suggestion

```